### PR TITLE
Display image after pagination

### PR DESCRIPTION
**Pull request by matthias-ronge**
Fixes: https://github.com/kitodo/kitodo-production/issues/563#issuecomment-281389951

### CODE REVIEW

File: Goobi/src/de/sub/goobi/metadaten/Metadaten.java [hunk 1]
```diff
@@ -1230,6 +1230,11 @@ public String KnotenAdd() throws TypeNotAllowedForParentException, TypeNotAllowe
 	 * @throws SwapException
 	 */
 	public String createPagination() throws TypeNotAllowedForParentException, IOException, InterruptedException, SwapException, DAOException {
+		createPagination(true);
+		return "";
+	}
+
+	private void createPagination(boolean bildErmitteln) throws TypeNotAllowedForParentException, IOException, InterruptedException, SwapException, DAOException {
```
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/metadaten/Metadaten.java]: It is possible to use a English named variable name?

File: Goobi/src/de/sub/goobi/metadaten/Metadaten.java [hunk 2]
```diff
@@ -1262,7 +1267,11 @@ public String createPagination() throws TypeNotAllowedForParentException, IOExce
 				}
 			}
 		}
-		return "";
+		
+		if (bildErmitteln) {
+			BildErmitteln(0);
+		}
+		return;
```
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/metadaten/Metadaten.java]: Is this last statement in this function necessary?

File: Goobi/src/de/sub/goobi/metadaten/Metadaten.java (hunk 1 above)
Comment (matthias-ronge) [inline Goobi/src/de/sub/goobi/metadaten/Metadaten.java]: I named it as the same of the function it calls.

File: Goobi/src/de/sub/goobi/metadaten/Metadaten.java (hunk 2 above)
Comment (matthias-ronge) [inline Goobi/src/de/sub/goobi/metadaten/Metadaten.java]: Yes can be removed.

**Pull request merged**